packages/node_modules/@webex/helper-image/src/index.js

```diff
@@ -43,21 +43,27 @@ export function updateImageOrientation(file, options = {}) {
 * @returns {Promise<ExifImage>}
 */
 export function readExifData(file, buf) {
-  return new Promise((resolve) => {
+  return new Promise(async (resolve) => {
```

Comment:
Not a big fan of mixing async with promises in this way, but, if it works, it works.

Reply:
`await`'ing is the suggested way to grab data hence the async